src/SFML/Main/MainWin32.cpp

```diff
@@ -41,6 +41,11 @@
 
 #include <windows.h>
 
+// Inform the Nvidia driver that this SFML application should
+// make use of the more powerful discrete GPU
+extern "C" {
+__declspec(dllexport) DWORD NvOptimusEnablement = 0x00000001;
```

Comment:
I'm also not convinced it's a good thing: someone could build an application that doesn't require huge GPU power and therefore could preserve battery life of a laptop by using the integrated GPU.